docs/gitbook/architecture/topology/examples.adoc

```diff
@@ -0,0 +1,39 @@
+= Examples
+
+By combining these molecules, you can achieve even more privacy and
+throughput gain.
+
+
+== The Hierarchical Throughput Molecule
+
+In this model, a single layer of hub and spoke does not supply enough
```

Comment:
This first sentence does not make sense. Why are we talking about a model that doesn't supply enough throughput? I think you are presenting the reason for the model, without saying so.